tests/components/switcher_kis/test_cover.py

```diff
@@ -34,10 +34,37 @@
 
 
 @pytest.mark.parametrize(
-    ("device", "entity_id"),
+    (
+        "device",
+        "entity_id",
+        "cover_id",
+        "position_open",
+        "position_close",
+        "direction_open",
+        "direction_close",
+        "direction_stop",
+    ),
     [
-        (DEVICE, ENTITY_ID),
-        (DEVICE2, ENTITY_ID2),
+        (
+            DEVICE,
+            ENTITY_ID,
+            0,
+            [77],
+            [0],
+            [ShutterDirection.SHUTTER_UP],
+            [ShutterDirection.SHUTTER_DOWN],
+            [ShutterDirection.SHUTTER_STOP],
+        ),
+        (
+            DEVICE2,
+            ENTITY_ID2,
+            0,
+            [77],
+            [0],
+            [ShutterDirection.SHUTTER_UP],
+            [ShutterDirection.SHUTTER_DOWN],
+            [ShutterDirection.SHUTTER_STOP],
+        ),
```

Comment:
Please remove this change, you added here parameters which are not changing (I understand they are planned for a future PR, but they are not relevant here). From what I understand only the `device` and `entity_id` are changing (same as before this change).

Reply:
It is done.